bin/lib/ce.py

```diff
@@ -11,10 +11,11 @@
 import sys
 import tempfile
 import time
-from argparse import ArgumentParser
 from collections import defaultdict
-from pprint import pprint
+from pprint import pformat
+from typing import TextIO, Optional, Dict, Sequence
 
+import click as click
```

Comment:
could this not just be `import click`?

Reply:
Erk yes! I have no idea what happened there. Fixed :)